packages/wallet/frontend/src/components/cards/PaymentPointerCard.tsx

```diff
@@ -51,20 +60,28 @@ PaymentPointerCardButton.displayName = 'PaymentPointerCardButton'
 
 export const PaymentPointerCard = ({
   paymentPointer,
+  isWM,
```

Comment:
```suggestion
  isWM = false,
```

Reply:
Not needed anymore, if I am always using same [accountId] page to come to the PPCards component, isWM will always have value